### PR TITLE
[Rust] Changed 'UUID' handling into 'uuid' crate

### PR DESCRIPTION
**Pull request by taufik-rama**
Moved from #12419

### CODE REVIEW

File: samples/client/petstore/rust/reqwest/petstore-async/.openapi-generator/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-6.1.0-SNAPSHOT
\ No newline at end of file
+6.0.0-SNAPSHOT
```
Comment (wing328) [inline samples/client/petstore/rust/reqwest/petstore-async/.openapi-generator/VERSION:1]: looks like you didn't create this change based on the latest master.

can you please pull the latest master into your branch and update the samples again? thanks.

**Comment by taufik-rama** [comment]
hmm, I've re-run `$ ./bin/generate-samples.sh bin/configs/rust-*` after pulling `master` but it seems to still be `6.0.0`

**Comment by wing328** [comment]
You need to pull the latest master to your branch first

**Comment by taufik-rama** [comment]
yeah, I've pulled the latest master

```shell
taufik-rama@web:~/Programming/web/openapi-generator$ git status
On branch rust-uuid-crate-2
Your branch is up to date with 'me/rust-uuid-crate-2'.

nothing to commit, working tree clean
```

```shell
taufik-rama@web:~/Programming/web/openapi-generator$ git remote -v
me      git@github.com:taufik-rama/openapi-generator (fetch)
me      git@github.com:taufik-rama/openapi-generator (push)
origin  ssh://git@github.com/OpenAPITools/openapi-generator (fetch)
origin  ssh://git@github.com/OpenAPITools/openapi-generator (push)
```

```
taufik-rama@web:~/Programming/web/openapi-generator$ git pull origin master 
remote: Enumerating objects: 65, done.
remote: Counting objects: 100% (65/65), done.
remote: Compressing objects: 100% (17/17), done.
# stuff...
```

```
taufik-rama@web:~/Programming/web/openapi-generator$ ./bin/generate-samples.sh bin/configs/rust-*
# START SCRIPT: ./bin/generate-samples.sh
This script generates all configs under bin/configs by default.
You may generate a targeted script or set of scripts using glob patterns.

For example:
    ./bin/generate-samples.sh bin/configs/java-*
# stuff...
```

No changes for the `samples/` (only commits from `git pull`)
```
taufik-rama@web:~/Programming/web/openapi-generator$ git status
On branch rust-uuid-crate-2
Your branch is ahead of 'me/rust-uuid-crate-2' by 3 commits.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
```

**Comment by wing328** [comment]
Before you ran `./bin/generate-samples.sh bin/configs/rust-*`, did you run `mvn clean package` to rebuild the JAR?

**Comment by taufik-rama** [comment]
ah got it, thanks!

**Comment by wing328** [comment]
tested locally and the result is good:
```
running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running tests/pet_tests.rs (/Users/williamcheng/Code/openapi-generator/samples/client/petstore/rust/target/debug/deps/pet_tests-65af8d2a8a399d08)

running 1 test
test test_pet ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.88s

   Doc-tests petstore-reqwest

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.833 s
[INFO] Finished at: 2022-07-19T11:00:38+08:00
[INFO] ------------------------------------------------------------------------
```

**Pull request merged**